x-pack/plugin/sql/src/test/java/org/elasticsearch/xpack/sql/action/BasicFormatterTests.java

```diff
@@ -40,14 +42,18 @@
     public void testFormatWithHeader() {
         String[] result = formatter.formatWithHeader(firstResponse.columns(), firstResponse.rows()).split("\n");
         assertThat(result, arrayWithSize(4));
-        assertEquals("      foo      |         bar          |15charwidename!|superduperwidename!!!|      baz      |"
-                + "          date          ", result[0]);
-        assertEquals("---------------+----------------------+---------------+---------------------+---------------+"
-                + "------------------------", result[1]);
-        assertEquals("15charwidedata!|1                     |6.888          |12                   |rabbit         |"
-                + "1953-09-02T00:00:00.000Z", result[2]);
-        assertEquals("dog            |1.7976931348623157E308|123124.888     |9912                 |goat           |"
-                + "2000-03-15T21:34:37.443Z", result[3]);
+        logger.info(result[0]);
```

Comment:
Do we need those?

Reply:
We don't. Not paying attention. Thanks for catching.